ci.sh

```diff
@@ -95,22 +95,7 @@ test_dmd() {
 # test dub package
 test_dub_package() {
     source ~/dlang/*/activate # activate host compiler
-    # GDC's standard library is too old for some example scripts
-    if [[ "${DMD:-dmd}" =~ "gdmd" ]] ; then
-        echo "Skipping DUB examples on GDC."
-    else
-        local abs_build_path="$PWD/$build_path"
-        pushd test/dub_package
-        for file in *.d ; do
-            # build with host compiler
-            dub --single "$file"
-            # build with built compiler (~master)
-            DFLAGS="-de" dub --single --compiler="${abs_build_path}/dmd" "$file"
-        done
-        popd
-        # Test rdmd build
-        "${build_path}/dmd" -version=NoBackend -version=GC -version=NoMain -Jgenerated/dub -Jres -Isrc -i -run test/dub_package/frontend.d
-    fi
+    test_dub_package_impl
```

Comment:
It doesn't matter where it's sourced, since the sourced script contains a function. The function needs to be called anyway.

Reply:
Yes, but it matters whether you source or call it. Calling is imho better because it avoids any potential issues with the namespace and allows to reproduce it easier.

Reply:
Ah, you mean calling the whole script? I can do that.